src/main/java/bowling/domain/Pin.java

```diff
@@ -35,17 +35,18 @@ public boolean isStrike() {
     }
 
     public boolean isSpare(Pin secondPins) {
+        validationPinCount(totalFallenPins(secondPins));
         if (isStrike()) {
             return false;
         }
         return totalFallenPins(secondPins) == MAXIMUM_PIN_COUNT;
     }
 
-    private int totalFallenPins(Pin secondPins) {
+    public int totalFallenPins(Pin secondPins) {
         return pins + secondPins.count();
```

Comment:
같은 객체이므로 getter를 사용하지 않고 바로 접근이 가능합니다 :)
```suggestion
        return pins + secondPins.pins;
```

Reply:
아! 하긴 같은 객체니깐 궂이 getter를 안써도 되겟네요!
이부분 수정하였습니다!